docs/templates/finops-hub.md

```diff
@@ -37,6 +37,8 @@ On this page:
 > 🆕 _Add the following parameters:_
 >
 > - **exportScopes** (array) – Optional. List of scope IDs to create exports for.
+> - **msCmExports_dataRetentionInDays** (int) – Optional. Number of days of cost data to retain. 0 = will delete after transforming. Default: 0 days.
+> - **ingestion_dataRetentionInMonths** (int) – Optional. Number of months of cost data to retain. Default: 13 months.
```

Comment:
@aromano2 I missed this in the last spec update. Can you add this to a new PR when you close the current one? Please resolve this comment to ack.